var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -84,6 +85,11 @@ def configure_args(self):
                 "--with-xml-prefix=%s" % spec['libxml2'].prefix,
                 "--with-binutils=%s" % spec['binutils'].prefix]
 
+        if '^intel-oneapi-mpi' in spec:
+                mpiroot = spec['mpi'].prefix
+                args += ["--with-mpi-headers=%s" % os.path.join(mpiroot, "mpi/latest/include")]
+                args += ["--with-mpi-libs=%s" % os.path.join(mpiroot, "mpi/latest/lib/release")]
```

Comment:
Does using `spec['mpi'].headers.directories[0]` work? The `oneapi-*` packages define `libs`/`headers` functions that may update in the future (i.e. if later versions of `intel-oneapi-mpi` change the location of these headers/libs) so that may avoid problems.

Reply:
Yes, this seems to work. Updated PR.